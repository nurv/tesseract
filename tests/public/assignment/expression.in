1
1 + 2
123 * 456
"Foo"
"Foo" + "Bar"
Math.sin(Math.PI/2)
