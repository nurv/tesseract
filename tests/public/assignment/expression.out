> 1
> 3
> 56088
> "Foo"
> "FooBar"
> 1.0
> 